hnn_core/viz.py

```diff
@@ -1255,19 +1264,40 @@ def plot_laminar_csd(times, data, contact_labels, ax=None, colorbar=True,
         The matplotlib figure handle.
     """
     import matplotlib.pyplot as plt
+    from scipy.interpolate import RectBivariateSpline
+
     if ax is None:
         _, ax = plt.subplots(1, 1, constrained_layout=True)
 
-    im = ax.pcolormesh(times, contact_labels, np.array(data),
-                       cmap="jet_r", shading='auto')
-    ax.set_title("CSD")
+    if sink[0] == 'b':
+        cmap = "jet"
+    elif sink[0] == 'r':
+        cmap = "jet_r"
+
+    if interpolation == 'spline':
+        # create interpolation function
+        interp_data = RectBivariateSpline(times, contact_labels, data.T)
+        # increase number of contacts
+        new_depths = np.linspace(contact_labels[0], contact_labels[-1],
+                                 contact_labels[-1] - contact_labels[0])
+        # interpolate
+        data = interp_data(times, new_depths).T
+    elif not interpolation:
```

Comment:
```suggestion
    elif interpolation is None:
```

in python `None` is a specific data type. It's not a `boolean` that can be True or False. Better to be explicit, otherwise the user may pass `interpolation=False` and that may also work but it is unintentional